src/components/sidebar/Sidebar.js

```diff
@@ -240,11 +240,58 @@ Content.defaultProps = {
 export default class Sidebar extends PureComponent {
```

Comment:
I started to develop inside `ChannelMembers`, but then understood that scroll controlled from `Sidebar `. Because except members list, scroll view contains other things like the name of the channel, buttons to invite somebody or leave the channel, etc.. If I'll develop this inside `ChannelMembers` we will receive the double scroll. That's not a good idea I think.

If I somehow can I can write `onScroll` method inside `ChannelMembers` and execute this method, inside onScroll event in `Sidebar` that would be perfect :D But I don't know how I implement this:(

Reply:
Ok, I found a solution - I can do ref to this `div` to `ChannelMembers `. Is it ok? What do you think @nikgraf ?